easybuild/easyconfigs/q/Qt/Qt-4.8.7-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Qt'
+version = '4.8.7'
```

Comment:
Please consider defining the variable `shortversion = 4.8` to be used in the download link to reduce errors when updating the recipe in the future